src/main/java/org/sonarqube/gradle/SonarQubeTask.java

```diff
@@ -27,11 +27,10 @@
 import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.Properties;
-import org.gradle.api.DefaultTask;
+import org.gradle.api.internal.ConventionTask;
```

Comment:
Dependency on internal API?

Reply:
good catch

Reply:
actually convention mapping is internal. Currently we already import org.gradle.api.internal.ConventionMapping.